VTEX - Master Data API - v2.json

```diff
@@ -120,6 +120,264 @@
+        "/api/dataentities/AD/documents": {
+            "post": {
+                "tags": ["Addresses"],
+                "summary": "Create new customer address",
+                "description": "Create new customer address. \r\n\r\n> You can use this request to create customer addresses according to any `AD` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\/\/developers.vtex.com\/vtex-rest-api\/docs\/master-data-schema-lifecycle).",
+                "operationId": "CreateNewCustomerAddress",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "$ref": "#/components/parameters/_schema"
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/createUpdateAddressRequests"
+                            }
+                        }
+                    },
+                    "required": true
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/documentResponse"
+                                },
+                                "example": {
+                                    "Id": "AD-b818cbda-e489-11e6-94f4-0ac138d2d42e",
+                                    "Href": "http://api.vtex.com/my-store-name/dataentities/AD/documents/b818cbda-e489-11e6-94f4-0ac138d2d42e"
+                                }
+                            }
+                        }
+                    }
+                },
+                "deprecated": false
+            }
+        },
+        "/api/dataentities/AD/documents/{id}": {
+            "patch": {
+                "tags": ["Addresses"],
+                "summary": "Update customer address",
+                "description": "Partially update a customer address.\r\n\r\n> You can use this request to update customer addresses according to any `AD` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\\\/\\\/developers.vtex.com\\\/vtex-rest-api\\\/docs\\\/master-data-schema-lifecycle).",
```

Comment:
```suggestion
                "description": "Partially updates a customer address.\r\n\r\n> You can use this request to update customer addresses according to any `AD` schema. Because of this, you are not restricted to using the fields exemplified below in your requests. But you should be aware of the fields allowed or required for the schemas you are using. Learn more about how to use [Master Data v2 schemas](https:\\\/\\\/developers.vtex.com\\\/vtex-rest-api\\\/docs\\\/master-data-schema-lifecycle).",
```

Reply:
Esse link tá certo mesmo?